public/js/views/AlbumReleasesView.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var Backbone = require('backbone');
+
+module.exports = Backbone.View.extend({
+
+});
```

Comment:
Same thing here - this will  never run in the browser without a build step. Also, this is an empty view implementation, meaning it won't actually do anything.